Rakefile

```diff
@@ -0,0 +1,15 @@
+# -*- ruby -*-
+
+require 'hoe'
+require './lib/pmdtester/parsers/options.rb'
+
+Hoe.spec 'pmdtester' do
+  self.version = PmdTester::Options::VERSION
+
+  self.author  = 'Binguo Bao'
+  self.email   = 'djydewang@gmail.com'
+
```

Comment:
I think, with the option "history_file" we can make the changelog also using markdown. I don't have a strong preference for the name of the changelog (PMD uses release_notes.md somewhere in the docs, eclipse plugin uses ReleaseNotes.md).

I'm not sure about the correct syntax, maybe:

```
  self.history_file = 'History.md'
```

Actually, [reading the source of hoe yields](https://github.com/seattlerb/hoe/blob/master/lib/hoe.rb#L650): It will use "History.txt" or "History.md", whichever file exists. If none exists, then "History.txt" is used by default. So, if we stay with "History.md", then you just need to rename the file, if we choose a different filename entirely, then we need to configure hoe.

Reply:
> Actually, reading the source of hoe yields:

I didn't find any documents on how to set up "history file", so it seems that reading the source code is also a good choice:)